os/kernel/binary_manager/binary_manager_bootparam.c

```diff
@@ -141,28 +142,31 @@ int binary_manager_scan_bootparam(binmgr_bpinfo_t *bp_info)
 		/* Read bootparam data */
 		ret = read(fd, (FAR uint8_t *)bootparam, BOOTPARAM_SIZE);
 		if (ret != BOOTPARAM_SIZE) {
-			printf("[Bootparam Scanning] Failed to read bootparam, errno %d\n", errno);
+			bmdbg("Fail to read BP, errno %d\n", errno);
 			continue;
 		} else if (!is_valid_bootparam(bootparam)) {
+			bmdbg("BP%d is invalid\n", bp_idx);
 			continue;
 		}
 
+		bmdbg("BP%d is valid\n", bp_idx);
```

Comment:
This log might not be needed? as all conditions covered with logs above

Reply:
I added it for readability like boot loader logs because it always checks both BPs.